FESS-GUI/setpassworddialog.cpp

```diff
@@ -13,6 +14,7 @@ SetPasswordDialog::SetPasswordDialog(QWidget *parent) :
     ui(new Ui::SetPasswordDialog)
 {
     ui->setupUi(this);
+    qsrand(time(NULL));
```

Comment:
Nice. On a broader note, I don't agree with moving this from the mainwindow constructor to the setpassworddialog constructor. qrand() is used in GetRandomString, which is not a member of setpassworddialog. So it isn't guaranteed that a setpassworddialog will be constructed before that funtion is used (if it is called from outside the class). Furthermore, qsrand only needs to be set up once; setting it up every time a setpassworddialog is created is unneccessary. If anything, qsrand(time(NULL)) should be moved to main(), the entry point of the program.